src/resources/micromarkdown-js.html

```diff
@@ -0,0 +1,6 @@
+<script
```

Comment:
Move this file and it's dependencies and put it in the `static` folder inside Home Assistant. The polymer dir and bower_components dir are not included in production.